ui/components/component-library/banner-base/banner-base.types.ts

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import type {
+  PolymorphicComponentPropWithRef,
+  StyleUtilityProps,
+} from '../box';
+
+export interface BannerBaseStyleUtilityProps extends StyleUtilityProps {
+  /**
+   * The title of the BannerBase
+   */
+  title?: string;
+  /**
+   * Additional props to pass to the `Text` component used for the `title` text
+   */
+  titleProps?: any;
```

Comment:
`titleProps?: TextProps<'span'>` might be wrong but this would be better, right?